cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -74,7 +74,6 @@ func NewAddCmd(o *AddOrUpdateOptions, flagsFactory cmdcore.FlagsFactory) *cobra.
 		DefaultTimeout:   5 * time.Minute,
 	})
 
-	// For `add` command create option will always be true
 	o.CreateRepository = true
```

Comment:
So are we using this variable to determine whether or not the run command should update or add? Seems like we could do this better, but definitely not blocking.

We should however change the check on line 138 to use the `errors` library in this PR imo.
Or do yopu wanna do this in a latter PR?

Reply:
I wanted to keep the changes in this PR to a minimum. We can definitely do some refactoring in later PRs which do not change the behaviour for users.

Reply:
These will be going in with the next PR as discussed on slack